lib/config.js

```diff
@@ -58,31 +10,17 @@ var bowerConfig = {
     // TODO: Use the official nunjucks and not a modified one.
     // 'nunjucks/browser/nunjucks-slim.js': LIB_DEST_PATH,
     'underscore/underscore.js': LIB_DEST_PATH,
-    'marketplace-core-modules/tests/**/*.js': CORE_DEST_PATH + 'tests',
-    'marketplace-core-modules/tests/tests.html': 'src/templates/commonplace/'
 };
-CORE_MODULES.forEach(function(module) {
-    if (CORE_MODULES_ROOT_JS.indexOf(module) !== -1) {
-        // Modules that go into the root JS path (for some reason).
-        bowerConfig[CORE_SRC_PATH + module + '.js'] = JS_DEST_PATH;
-        return;
-    }
-    bowerConfig[CORE_SRC_PATH + module + '.js'] = CORE_DEST_PATH;
-});
-CORE_VIEWS.forEach(function(view) {
-    bowerConfig[CORE_SRC_PATH + view + '.js'] = CORE_DEST_PATH + 'views/';
-});
 
 // Build require config, to be used in development and AMD optimizers.
 var requireConfig = {
     enforceDefine: true,
     paths: {
+        'core': 'lib/core',
```

Comment:
I figured I'd leave it in the parent app in case they don't want to use mcm but it makes sense to move it here. No harm in extra rewrites that do nothing.